soc/ite/ec/it8xxx2/Kconfig

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_ITE_IT8XXX2
+	select CPU_HAS_FPU if "$(ZEPHYR_TOOLCHAIN_VARIANT)" != "zephyr" || RISCV_ISA_EXT_M
+	select SOC_FAMILY_ITE_EC
```

Comment:
`	select SOC_FAMILY_ITE_EC` should be in `Kconfig.soc`

Reply:
ok done